src/main/java/edu/kis/powp/jobs2d/features/DrawerOdometerFeature.java

```diff
@@ -0,0 +1,31 @@
+package edu.kis.powp.jobs2d.features;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+import java.util.logging.Logger;
+
+public class DrawerOdometerFeature implements Job2dDriver {
+    Logger logger = Logger.getLogger("global");
+    private Double
+            lineA,
+            lineB,
+            positionDistance = 0.0,
+            operateDistance = 0.0;
+
+    @Override
+    public void setPosition(int i, int i1) {
+        lineA = Double.valueOf(i);
+        lineB = Double.valueOf(i1);
+        double hypotenuse = Math.sqrt(Math.pow(lineA, 2) + Math.pow(lineB, 2));
+        this.positionDistance = this.positionDistance + hypotenuse;
+        logger.info("Head distance: " + this.positionDistance);
```

Comment:
operateTo should count in "Head distance" (the head is moved). Log both head an op. distances together (without code duplication). 
BTW: logging should be optional